Classes/Session/TSSTPageView.m

```diff
@@ -302,6 +302,9 @@ - (void)drawRect:(NSRect)aRect
 	NSImageInterpolation interpolation = [self inLiveResize] || scrollKeys ? NSImageInterpolationLow : NSImageInterpolationHigh;
 	[[NSGraphicsContext currentContext] setImageInterpolation: interpolation];
 	
+	self.layer.sublayers = nil;
+	self.layer.backgroundColor = [[NSColor blackColor]CGColor];
```

Comment:
Is there a reason why the background color isn't set from the defaults via `TSSTBackgroundColor`?

Reply:
Nope. I have very little knowledge of the whole code-base, and ignored the existence of TSSTBackground Color. It should actually be:
```
self.layer.sublayers = nil;
	
NSUserDefaults * defaults = [NSUserDefaults standardUserDefaults];
NSColor * color = [NSKeyedUnarchiver unarchiveObjectWithData: [defaults valueForKey: TSSTBackgroundColor]];
self.layer.backgroundColor = [color CGColor];
```

I will submit a new pull request with this correction.